README.md

```diff
@@ -166,14 +167,16 @@ For example
 ### ls example in detail
 
 ```
-> <varint-length>ls<newline>
-< <varint-length><varint-of-list-of-protos-size-in-bytes><varint-number-of-protocols><newline>
-< <varint-length><protocol><newline>
-# ...
-< <varint-length><protocol><newline>
+> <varint-ls-length>ls\n
```

Comment:
Note: ls isn't a proper protocol and probably shouldn't be listed.

Reply:
In the response? Yeah that makes sense. Shall I add that note?

Reply:
I'd just remove it. We shouldn't list it as a protocol.

Reply:
...but this is the example where the local node sends `ls` and the remote then responds.

Reply:
Someday I'll learn how to read.